locations/spiders/anthonys.py

```diff
@@ -0,0 +1,40 @@
+import html
+import re
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+
+class AnthonysSpider(scrapy.Spider):
+    name = "anthonys"
+    item_attributes = {"brand": "Anthony's Coal Fired Pizza", "country": "US"}
+    allowed_domains = ["wp.acfp.com"]
+    start_urls = ["https://wp.acfp.com/wp-json/wp/v2/locations?per_page=100"]
+
+    def parse(self, response):
+        for store in response.json():
+            address = re.match(
+                r"<p>(\d+) ([-. \w]+)<\/p>\n<p>([ \w]+), (\w{2}) (\d+)<\/p>",
+                store["acf"]["page"]["info"]["address"],
+            )
+            item = GeojsonPointItem(
+                {
+                    "lat": store["acf"]["coords"]["latitude"],
+                    "lon": store["acf"]["coords"]["longitude"],
+                    "name": html.unescape(store["title"]["rendered"]),
+                    "addr_full": html.unescape(store["acf"]["page"]["info"]["address"]),
```

Comment:
This looks to be pulling in a bunch of extra whitespace. Can you trim/strip?

For example:
```
"
7776 Norfolk Avenue

\n
Bethesda, MD 20814

\n",
```